app/assets/stylesheets/notes.scss

```diff
@@ -0,0 +1,77 @@
+table {
+  margin-left: 10%;
+  margin-right: 10%;
+  border-spacing: 0;
+  width: 80%;
+  border: 1px solid #ddd;
+}
+
+th, td {
+  text-align: left;
+  padding: 16px;
+}
+
+tr:nth-child(even) {
+  background-color: #edf3ff;
+}
+
+td{
+  color: black;
+}
+
+.new {
+  font-size: 16px;
+  margin-left: 20%;
+  padding: 20px;
+}
+
+#search-box{
```

Comment:
Use one case convention for IDs and classes consistently (hyphens or camel case).